src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlCommand.cs

```diff
@@ -617,7 +617,7 @@ public override string CommandText
             get
             {
                 string value = _commandText;
-                return ((null != value) ? value : ADP.StrEmpty);
+                return ((null != value) ? value : ADP.s_strEmpty);
```

Comment:
Can we just get rid of empty string constants in favour of using "" or string.Empty?

Reply:
I wanted to remove it if there wasn't the [comment](https://github.com/dotnet/SqlClient/pull/1306/files#diff-28fd361d833bb12ffb3d5462abcd11f1fa22c9157c923510e76bf1c4c9c0932eR512) over its definition. My question is how's it possible for who has an answer!?

from the code:
```
 // security issue, don't rely upon public static readonly values
 internal static readonly string s_strEmpty = ""; // String.Empty
```

Reply:
All instances of "" will be interned by the runtime, if you have one you have all of them. The only way to create a different one would be to to use string.Create() in a way that prevents the interned string check. So even though we intend to have our own copy of the constant we won't, we'll still get the exact same one as is publicly available.

Moreover the only ways to change a string are unsafe and unsupported so if you do any of them you put the entire runtime in a state it isn't going to be able to cope with. If you somehow changes "" to be non-empty you would break many many other pieces of software.

Reply:
Done. I preferred to use `""` to avoid the suspected security issue.